FirebaseCore/CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # FirebaseCore 7.7.0
 - [changed] Deprecated FirebaseMLModelInterpreter and FirebaseMLVision.
 - [added] Introduced FirebaseMLModelDownloader.
-- [fixed] Fixed missing doc comment in `FirebaseVersion()` (#7506).
+- [fixed] Fixed missing doc comment in `FirebaseVersion()`. (#7506)
+- [changed] Minimum required Xcode version for Zip and Carthage distributions changed to 12.2 (was 12.0).
+- [added] The zip distribution now includes Catalyst simulator slices. (#7007)
```

Comment:
Nit: it's `Catalyst arm64 simulator slices` - the existing zip does support it but just not for Apple silicon.